.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+.idea
```

Comment:
This needs to be in your global `.gitignore`, it is not something specific to this repo.